woocommerce/woocommerce-android
# Issue/3207 add attribute part5

This PR continues the attributes & terms work but adding the ability to save all attribute & term changes in a single call when exiting the add terms fragment (thanks to the FluxC work by @ThomazFB ). Note that this isn't the fragment we'll eventually save changes from, but for now this gives us the ability to test the logic without backing out through multiple fragments.

This PR also adds a skeleton to the global attributes shown on the term fragment.

To test:

* Add global attribute terms and verify they're saved to the backend when leaving the terms fragment
* Likewise for local attributes

Update release notes:
